docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+[[release-notes-8.0.0]]
+== {es} version 8.0.0
+
+coming::[8.0.0]
+
+The following list are changes in 8.0.0 as compared to 7.17.0, and combines
+release notes from the 8.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
+Also see <<breaking-changes-8.0,Breaking changes in 8.0>>.
+
+[[breaking-8.0.0]]
+[float]
+=== Breaking changes
+
+Aggregations::
+* Percentiles aggregation: disallow specifying same percentile values twice {es-pull}52257[#52257] (issue: {es-issue}51871[#51871])
+* Remove Adjacency_matrix setting {es-pull}46327[#46327] (issues: {es-issue}46257[#46257], {es-issue}46324[#46324])
+* Remove `MovingAverage` pipeline aggregation {es-pull}39328[#39328]
+* Remove deprecated `_time` and `_term` sort orders {es-pull}39450[#39450]
+* Remove deprecated date histo interval {es-pull}75000[#75000]
+
+Allocation::
+* Breaking change for single data node setting {es-pull}73737[#73737] (issues: {es-issue}55805[#55805], {es-issue}73733[#73733])
```

Comment:
This doesn't describe the change. How about something along the lines of:
```suggestion
* Require single data nodes to respect disk watermarks {es-pull}73737[#73737] (issues: {es-issue}55805[#55805], {es-issue}73733[#73733])
```

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c